src/examples/java/io/nats/examples/NatsJsManage.java

```diff
@@ -0,0 +1,132 @@
+            StreamConfiguration sc1 = StreamConfiguration.builder()
+                    .name(STREAM1)
+                    .storageType(StreamConfiguration.StorageType.Memory)
+                    .subjects(STRM1SUB1, STRM1SUB2)
+                    .build();
+            js.addStream(sc1);
+
+            StreamInfo si = js.streamInfo(STREAM1);
+            printObject(si);
+
+            action("Configure And Add Stream 2");
+            StreamConfiguration sc2 = StreamConfiguration.builder()
+                    .name(STREAM2)
+                    .storageType(StreamConfiguration.StorageType.Memory)
+                    .subjects(STRM2SUB1, STRM2SUB2)
+                    .build();
+            js.addStream(sc2);
+
+            si = js.streamInfo(STREAM2);
+            printObject(si);
+
+            action("Delete Stream 2");
+            js.deleteStream(STREAM2);
+
+            action("Delete Non-Existent Stream");
+            try {
+                js.deleteStream(STREAM2);
+            }
+            catch (IllegalStateException ise) {
+                System.out.println(ise.getMessage());
+            }
+
+            action("Update Non-Existent Stream ");
+            try {
+                StreamConfiguration non = StreamConfiguration.builder()
+                        .name(STREAM2)
+                        .storageType(StreamConfiguration.StorageType.Memory)
+                        .subjects(STRM2SUB1, STRM2SUB2)
+                        .build();
+                js.updateStream(non);
+            }
+            catch (IllegalStateException ise) {
+                System.out.println(ise.getMessage());
+            }
+
+            action("Configure And Add Consumer");
+            ConsumerConfiguration cc = ConsumerConfiguration.builder()
+                    .deliverSubject("strm1-deliver")
+                    .durable("GQJ3IvWo")
+                    .build();
+            ConsumerInfo ci = js.addConsumer(STREAM1, cc);
+            printObject(ci);
+            si = js.streamInfo(STREAM1);
+            printObject(si);
+
+            action("Make And Use Subscription");
+            SubscribeOptions so = SubscribeOptions.builder()
+                    .pullDirect(STREAM1, "GQJ3IvWo", 10)
+//                    .configuration(STREAM1, cc)
+                    .build();
+            printObject(so);
+
+            si = js.streamInfo(STREAM1);
+            printObject(si);
+
+            JetStreamSubscription sub = js.subscribe(STRM1SUB1, so);
+            printObject(sub);
+
+            action("List Consumers");
+            ConsumerLister lister = js.newConsumerLister(STREAM1);
+            printObject(lister);
+        }
+        catch (Exception exp) {
+            action("EXCEPTION!!!");
+            exp.printStackTrace();
+        }
+    }
+
+    private static void action(String label) {
+        System.out.println("================================================================================");
```

Comment:
Looks like an issue with spacing?

Reply:
Just debug stuff using to show what's going on